src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -304,6 +319,20 @@ $(BashLinkerTestCleanupCmds)
       <![CDATA[
 cd WasmApp
 ./run-v8.sh $(MsBuildProjectName)
+CLRTestExitCode=$?
+      ]]>
+      </BashCLRTestLaunchCmds>
+      <BashCLRTestLaunchCmds Condition="'$(CLRTestKind)' == 'BuildAndRun' And $(TargetOS) == 'Android' ">
+      <![CDATA[
+# run Android app
+AdbTool=$ANDROID_SDK_ROOT/platform-tools/adb
+$AdbTool kill-server
+$AdbTool start-server
+$AdbTool logcat -c" ContinueOnError="WarnAndContinue
+$AdbTool uninstall net.dot.$(MsBuildProjectName)" ContinueOnError="WarnAndContinue
+$AdbTool install apk/bin/$(MsBuildProjectName).apk
+$AdbTool shell am instrument -w net.dot.$(MsBuildProjectName)/net.dot.MonoRunner
+$AdbTool logcat -d -s DOTNET
```

Comment:
It should use xharness instead, I don't actually remember why the Android sample still uses raw `adb`

Reply:
I tried to use xharness. And there are compatible issues with runtime tests. One issue is that runtime tests don't generated testResults.xml files. Workarounds needs to be implemented for runtime tests, in order to use xharness on runtime tests.